src/style.css

```diff
@@ -1,3 +1,82 @@
+html {
+  /* stylelint-disable-next-line font-family-name-quotes */
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: normal;
+  text-transform: uppercase;
+}
+
+:root {
+  --color:#00ACDC;
+}
+
 body {
   margin: 0;
 }
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  height: 60px;
+  width: 100%;
+  padding: 0 50px;
+
+  box-sizing: border-box;
+}
+
+.nav_list {
+  padding: 0;
+  margin: 0;
+  height: 60px;
+  display: flex;
+  align-items: center;
+}
+
+.nav_item {
+  list-style: none;
+  font-size: 12px;
+  text-transform: uppercase;
+  font-weight: 500;
+  line-height: 60px;
+}
+
+.nav_item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav_link {
+  text-decoration: none;
+  color: black;
+  display: flex;
+}
+
+.is-active {
+  color: var(--color);
+  position: relative;
+}
+
+.is-active::after {
+  content: '';
+  position: absolute;
+  background-color: var(--color);
+  height: 4px;
+  width: 100%;
+  bottom: 0;
+  display: block;
+  /* stylelint-disable-next-line length-zero-no-unit */
+  right: 0;
+  left: 0;
+  border-radius: 8px;
+}
+
+.over:hover {
+  color: var(--color);
+}
```

Comment:
all your nav_link elements should have a hover effect, not just the one used in tests ;)